authenticity-collector/src/main.ts

```diff
@@ -2,16 +2,35 @@ import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
 import { ChannelSubscriptionService } from './services/channel-subscription/channel-subscription.service';
 import { CollectorIdentityService } from './services/collector-identity/collector-identity.service';
+import { SwaggerModule, DocumentBuilder, SwaggerDocumentOptions } from '@nestjs/swagger';
+import { Logger } from '@nestjs/common';
 
 async function bootstrap() {
 	const app = await NestFactory.create(AppModule);
-	await app.listen(process.env.port || 3000);
 
-	const collectorIdentityService = app.get(CollectorIdentityService);
-	await collectorIdentityService.checkCollectorIdentity();
+	const options: SwaggerDocumentOptions = {
+		operationIdFactory: (controllerKey: string, methodKey: string) => methodKey
+	};
 
-	const channelSubscription = app.get(ChannelSubscriptionService);
-	await channelSubscription.channelSubscription();
+	const config = new DocumentBuilder()
+		.setTitle('Authenticity-Collector-API-Reference')
+		.setDescription(
+			'Authenticity Collector is a microservice that saves the authentication proof of the devices of CityXChange. Furthermore it writes the sensor data, which is send by the device, to his own channel and sends it also to MPower.'
+		)
+		.setVersion('1.0')
+		.addTag('authenticity-collector')
+		.build();
+
+	const document = SwaggerModule.createDocument(app, config, options);
+	SwaggerModule.setup('/api/v1/authenticity-collector-docs', app, document);
+
+	await app.listen(process.env.PORT || 3000);
+	// const collectorIdentityService = app.get(CollectorIdentityService);
```

Comment:
can you please remove the commented code?

Reply:
This will be uncommented in final - yesterday the IS was down and I could not work on the swagger docs another way